ntc_templates/templates/index

```diff
@@ -30,6 +30,7 @@ alcatel_sros_show_router_mpls_lsp.textfsm, .*, alcatel_sros, sh[[ow]] router mpl
 alcatel_sros_show_service_id_base.textfsm, .*, alcatel_sros, sh[[ow]] serv[[ice]] id ba[[se]]
 alcatel_sros_show_service_sdp.textfsm, .*, alcatel_sros, sh[[ow]] service sdp
 alcatel_sros_show_system_cpu.textfsm, .*, alcatel_sros, sh[[ow]] system cpu
+alcatel_sros_show_lag_port.textfsm, .*, alcatel_sros, sh[[ow]] lag (\d+ )?port
```

Comment:
@bennnnnnnn
I don't have any Alcatel/Nokia SROS gear to poke at so I'm going to pose questions below.

After the `show` word, does `lag` always need spelled out fully? Or is there a shortcut of say `la`.

Similarly for `port` after the interface number, from the CLI is it possible to issue a shorter keyword?
(If yes `port` could be changed to accept abbreviations.)